### PR TITLE
Origin/docker compose apps

### PR DESCRIPTION
**Pull request by AinaArd**

### CODE REVIEW

File: src/main/java/com/github/javalab/javaiaas/DockerImage.java [hunk 1]
```diff
@@ -66,8 +66,15 @@ private String safeReadOutput() {
...
         if (process.isAlive()) {
             process.destroyForcibly();
         }
     }
+
+    public void runApp(String port) throws IOException {
+        List<String> command = asList("PORT=" + port +  " docker-compose up --build");
+        processBuilder.command(command);
+        process = processBuilder.start();
+    }
```
Comment (Dimanaux) [inline src/main/java/com/github/javalab/javaiaas/DockerImage.java:79]: I don't understand

Comment (Dimanaux) [reply src/main/java/com/github/javalab/javaiaas/DockerImage.java:79]: Why?

File: src/main/resources/requests.http [hunk 2]
```diff
@@ -45,34 +46,21 @@ GET http://localhost:8080/applications/1
...
+DELETE http://localhost:8080/applications/2
 Authorization: eyJhbGciOiJIUzUxMiJ9.eyJsb2dpbiI6ImFpbmEiLCJpZCI6MX0.tNUVLtXwKaH5x1whga7TQbuOpCEDAl3_T_V4v6Jm_9CAQBJc4X_GsEXzgU3BWwGOe2A8g81ji9XuvXreEMaEaQ
 
+###
 
-### Check port
-GET http://localhost:8080/port/8080
-Accept: application/json
-Authorization: eyJhbGciOiJIUzUxMiJ9.eyJsb2dpbiI6ImFpbmEiLCJpZCI6MX0.tNUVLtXwKaH5x1whga7TQbuOpCEDAl3_T_V4v6Jm_9CAQBJc4X_GsEXzgU3BWwGOe2A8g81ji9XuvXreEMaEaQ
+GET http://localhost:8080/run?app=1
```
Comment (Dimanaux) [inline src/main/resources/requests.http:63]: Very bad, never do that! GET is for getting information. Use PUT or POST!

**Review by Dimanaux — CHANGES_REQUESTED** [review]

File: src/main/java/com/github/javalab/javaiaas/DockerImage.java (hunk 1 above)
Comment (AinaArd) [reply src/main/java/com/github/javalab/javaiaas/DockerImage.java:79]: why run app so?

**Pull request closed without merging**